src/EnergyPlus/CrossVentMgr.cc

```diff
@@ -588,7 +596,7 @@ namespace CrossVentMgr {
 		// Droom the distance between the average point of the base surface of the airflow network Surface (if the base surface
 		// is a Window or Door it looks for the second base surface).
 		// Dstar is Droom corrected for wind angle
-		Wroom = Zone( ZoneNum ).Volume / Zone( ZoneNum ).FloorArea;
+		EvolveParaUCSDCV_Wroom = Zone( ZoneNum ).Volume / Zone( ZoneNum ).FloorArea;
```

Comment:
Same for Wroom, Aroom, NodeNum1, and NodeNum2.  They are always set before they are used, so no need to put them in clear_state.